easybuild/easyblocks/i/impi.py

```diff
@@ -141,6 +143,22 @@ def post_install_step(self):
             for script in [os.path.join(script_path, 'mpivars.sh') for script_path in script_paths]:
                 apply_regex_substitutions(os.path.join(self.installdir, script), regex_subs)
 
+        # Recompile libfabric if necessary
+        if impiver >= LooseVersion('2019.0.0') and self.cfg['rebuild_libfabric']:
+            if self.cfg['ofi_internal']:
+                os.chdir(os.path.join(self.installdir, 'libfabric'))
+                extract_file('src.tgz', './')
+                libfabric_installpath = os.path.join(self.installdir, 'intel64', 'libfabric')
+                cmds = [
+                    './configure --prefix=%s %s' % (libfabric_installpath, self.cfg['libfabric_configopts']),
+                    'make',
```

Comment:
Should we care about `make -j N` here, or is rebuilding `libfabric` quick enough that it doesn't matter too much?